.travis.yml

```diff
@@ -0,0 +1 @@
+language: dart
```

Comment:
you need to add the script to run the specs

Reply:
According with travis docs we don't need it
![captura de tela 2019-02-08 as 14 31 58](https://user-images.githubusercontent.com/10780132/52491461-53199480-2bae-11e9-9b30-6f7aab46e25b.png)

But fort some reason he don't find the tests, I'll update it